spec/models/conversation_visibilities_spec.rb

```diff
@@ -4,25 +4,22 @@
 
 require 'spec_helper'
 
-describe ConversationVisibility, :type => :model do
-  before do
-    @user1 = alice
-    @participant_ids = [@user1.contacts.first.person.id, @user1.person.id]
-
-    @create_hash = {
-      :author => @user1.person,
-      :participant_ids => @participant_ids,
-      :subject => "cool stuff",
-      :messages_attributes => [ {:author => @user1.person, :text => 'hey'} ]
+describe ConversationVisibility, type: :model do
+  let(:user1)           { alice }
+  let(:participant_ids) { [user1.contacts.first.person.id, user1.person.id] }
+  let(:create_hash) do
+    {
+      author: user1.person,
+      participant_ids: participant_ids,
+      subject: 'cool stuff',
+      messages_attributes: [{ author: user1.person, text: 'hey' }]
```

Comment:
Space inside { detected.<br>Space inside } detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.